src/Ng/Program.cs

```diff
@@ -27,22 +27,22 @@ public static void Main(string[] args)
                 Debugger.Launch();
             }
 
-            // Get arguments
-            var arguments = CommandHelpers.GetArguments(args, 0);
-
-            // Configure ApplicationInsights
-            ApplicationInsights.Initialize(CommandHelpers.GetApplicationInsightsInstrumentationKey(arguments));
-
             // Create an ILoggerFactory
             var loggerConfiguration = LoggingSetup.CreateDefaultLoggerConfiguration(withConsoleLogger: true);
             var loggerFactory = LoggingSetup.CreateLoggerFactory(loggerConfiguration);
 
             // Create a logger that is scoped to this class (only)
             _logger = loggerFactory.CreateLogger<Program>();
```

Comment:
Then a separate try catch with command line logging

---
In reply to: [71394658](https://github.com/NuGet/NuGet.Services.Metadata/pull/77#discussion_r71394658) [](ancestors = 71394658,71253697)